_posts/en/2020-05-20-Team-Topologies-Whitelisting-app.md

```diff
@@ -3,59 +3,61 @@ layout: post
 
-<!--more-->
-
 While these timelines are ahead of what is typically found in government environments, it is only a starting point. My personal goal has always been to match Google's ninety minutes to create a prototype for Google Glass (reference [How Google Works](https://www.goodreads.com/book/show/23158207-how-google-works)). If we were to have demo'd the application to the CIO after 90 minutes of having started work, the only remaining question would be to how to get that number down to 60 minutes. Never stop improving.
 
 ## Model
 
 The methodology outlined throughout this document is heavily borrowed from [Team Topologies](https://teamtopologies.com/?gclid=EAIaIQobChMI4N2R7fzM6AIV1f_jBx2o3ACIEAAYASAAEgL8DPD_BwE).
 
-![Compare curves]({{ site.baseurl }}/assets/images/TTBlog-Figure1.PNG)
+![Team Topologies model]({{ site.baseurl }}/assets/images/TTBlog-Figure1.PNG)
 _Figure 1_
 
 In a government context, the management model looked as follows
 
-![Compare curves]({{ site.baseurl }}/assets/images/TTBlog-Figure2.PNG)
+![Government Team Topologies Model]({{ site.baseurl }}/assets/images/TTBlog-Figure2.PNG)
 _Figure 2_
 
 It should be noted that in our case, the separation between the DTS platform team and the IT Strategy team was not as well defined as it appears here. For example, throughout the project the DTS resources attended the same standup meetings as the IT Strategy resources. Further more, the value stream aligned team was composed of programmers from the DTS team as well. The reason why we have represented the interactions between the teams above is twofold. Firstly, to represent how responsibilities would be split between teams if this solution were to scale. In our case, the application was simple enough, and the team small enough, to permit for a mixing of resources as we did. As one scales, this becomes less feasible (keeping in mind we want to reduce the cognitive load on team members, and keep teams as small as possible, while still being able to deliver the product). The second reason the interactions have been represented this way, is to communicate how the technologies interacted. In between the teams we see XaaS (Anything as a Service), because this is the terminology used in the book, Team Topologies, that I am heavily leaning upon. I have recently begun to use a different term -- termination point. That is required is a termination point of responsibilities. In the case of our team, it was the termination point between members of the team, in large projects, it would be the termination point between different teams. A termination point is where the responsibilities of one group are transferred to another.
```

Comment:
Personal pet peeve...for humans to be described as 'resources'.  perhaps "people from the DTS and IT strategy teams working on HR Whitelisting attended standup meetings together"

Reply:
Also, I think furthermore is one word?

Reply:
Updated

Reply:
is it true that having team members working on the same project wouldn't be able to attend standups together if the project was to scale?  Does scaling always mean larger teams?

Reply:
i'm not sure what is meant by this sentence "That is required is a termination point of responsibilities"

Reply:
Aye, if folks need to meet in person to use each others services, then they're not using the model properly.

Ie. We are using CDS's Notify service, and it didn't required them to attend any of our standups. That model can scale.

If our app was larger and required many different services to function, we shouldnt have standups with 20 people in them ...

Reply:
> i'm not sure what is meant by this sentence "That is required is a termination point of responsibilities"

That -> What

Pushed the fix